resources/todomvc/architecture-examples/svelte/public/todomvc-app-css.css

```diff
@@ -0,0 +1,379 @@
+html,
+body {
+	margin: 0;
+	padding: 0;
+}
+
+button {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	background: none;
+	font-size: 100%;
+	vertical-align: baseline;
+	font-family: inherit;
+	font-weight: inherit;
+	color: inherit;
+	-webkit-appearance: none;
+	appearance: none;
+	-webkit-font-smoothing: antialiased;
```

Comment:
I don't think these things to be applied only in WebKit/Firefox.

Reply:
btw the specific css here is coming from npm https://github.com/WebKit/Speedometer/pull/68/files#diff-5df0fe52b53cca989a5f8385d250b1ddbefa104733a1658bf1112320abb8a8d9R33, which appears to be a different version from some of the existing todomvc demos. If we want to fix this maybe we should fork that CSS into this repo and then update all the relevant package.json files / copies of the CSS to point to that. I'm not sure though, curious for thoughts from @housseindjirdeh